shared/chat/conversation/messages/wrapper/height-retainer/index.desktop.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react'
+import * as ReactDOM from 'react-dom'
+import {resolveRootAsURL} from '../../../../../desktop/resolve-root'
+import {urlsToImgSet} from '../../../../../common-adapters/icon.desktop'
+import {Box} from '../../../../../common-adapters'
+import {collapseStyles} from '../../../../../styles'
+import type {Props} from '.'
+
+const explodedIllustration = resolveRootAsURL('../images/icons/icon-shh-24.png')
+const explodedIllustrationUrl = urlsToImgSet({'24': explodedIllustration}, 24)
+
+type State = {
+  height: ?number,
+}
+class HeightRetainer extends React.Component<Props, State> {
+  state = {height: 17}
```

Comment:
These set the initial height for when this mounts on an already-exploded message